pkg/manager/impl/execution_manager.go

```diff
@@ -328,12 +358,28 @@ func (m *ExecutionManager) RelaunchExecution(
 	if executionSpec.Metadata == nil {
 		executionSpec.Metadata = &admin.ExecutionMetadata{}
 	}
+	var inputs *core.LiteralMap
+	if len(existingExecutionModel.UserInputsUri) > 0 {
+		inputs = new(core.LiteralMap)
```

Comment:
Interesting you used `new` here. Not a problem but generally in our codebase we have been doing `&core.LiteralMap{}`

Reply:
Ah yeah, I wanted to ensure that golang realized to place it on the heap...although i think by observing the pointer escaping, golang will automatically allocate on the heap anyway.  So i can meet convention.